### DISCUSSION TITLE
rsuite update to V.5.44.0 SassError: Only 3 elements allowed, but 5 were passed. in react typescript

### DISCUSSION THREAD

**Discussion by autbaramee** [rsuite/rsuite#3452] [Q&A] [answered]
![image](https://github.com/rsuite/rsuite/assets/93520382/156a8fa5-02a7-4105-b383-7d7f574db093)

ERROR in ./node_modules/rsuite/dist/rsuite.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[7].use[1]!./node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[7].use[2]!./node_modules/resolve-url-loader/index.js??ruleSet[1].rules[1].oneOf[7].use[3]!./node_modules/react-scripts/node_modules/sass-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[7].use[4]!./node_modules/rsuite/dist/rsuite.css)
Module build failed (from ./node_modules/react-scripts/node_modules/sass-loader/dist/cjs.js):
SassError: Only 3 elements allowed, but 5 were passed.
    ╷
345 │ @supports not (color: rgb(from white r g b)) {
    │                       ^^^^^^^^^^^^^^^^^^^^^
    ╵
  node_modules\rsuite\dist\rsuite.css 345:23  root stylesheet

**Comment by simonguo** [comment, marked as answer]
Hi @autbaramee , https://github.com/rsuite/rsuite/pull/3428  here we use CSS relative color syntax, which causes Sass to fail to parse.

https://github.com/sass/sass/issues/3673 there is a discussion about this in Sass, hope it helps.

**Reply by autbaramee** [reply]
Thank you. I switched from importing through Sass to using CSS instead.

**Reply by simonguo** [reply]
https://github.com/rsuite/rsuite/pull/3462 
I used the postcss plugin to be compatible with CSS relative color syntax. Please try the next version.
